examples/koa-http-server-ts/src/index.ts

```diff
@@ -1,5 +1,5 @@
-import Koa from 'koa';
-
+import * as Koa from 'koa';
```

Comment:
```suggestion
import Koa from 'koa';
```

Set `"esModuleInterop": true` in `tsconfig.json`